railties/lib/rails/generators/rails/encrypted_secrets/templates/config/secrets.yml.enc

```diff
@@ -0,0 +1,2 @@
+production:
+  secret_key_base: <%= SecureRandom.hex(64) %>
```

Comment:
It is optional. It will only be generated when people call `bin/rails secrets:setup` manually 😊